bundler.go

```diff
@@ -150,8 +152,16 @@ func New(c *Configuration) (b *Bundler, err error) {
 	}
 
 	// Paths that depends on the input path
-	b.pathBuild = strings.TrimPrefix(strings.TrimPrefix(b.pathInput, filepath.Join(os.Getenv("GOPATH"), "src")), string(os.PathSeparator))
-	b.pathResources = filepath.Join(b.pathInput, "resources")
+	if strings.HasPrefix(b.pathInput, filepath.Join(os.Getenv("GOPATH"), "src")) {
```

Comment:
Why did you add this `if` condition? If this is to allow using relative path and avoid the `TrimPrefix(os.PathSeparator)`, I'd rather you use `filepath.IsAbs()` method or event better the `filepath.Abs()` method

Reply:
For gop, a folder is a temp `GOPATH`, so the build path is relative to this temp `GOPATH`. This `if` will confirm the required build package is in the global `GOPATH`.

Reply:
ok but you're doing `if has prefix then trim prefix` which is the same as `trim prefix`. In this case I'd rather you remove the if condition. Or maybe there's something I'm missing out.